shap/benchmark/methods.py

```diff
@@ -88,6 +90,16 @@ def tree_gain(model, data):
     """
     return other.TreeGainExplainer(model).attributions
 
+def permutation_shap_1(model, data):
+    """ Permutation Explainer
+    """
+    return lambda X: PermutationExplainer(model.predict, data).shap_values(X, npermutations=1)
+
+def permutation_shap_10(model, data):
+    """ Permutation Explainer
```

Comment:
I'm wondering if we need to add docs here, eg model and data should ideally be documented.
It does seem like shap explainers are documented very well, for example:
https://github.com/slundberg/shap/blob/master/shap/explainers/_tree.py
I would think these utilities should have similar nice documentation, especially for public methods

Reply:
Will remove this because it is for testing and no longer valid.